src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -724,17 +747,14 @@ def add_error_to_status(self, message, error_code=Constants.PatchOperationErrorC
         if not message or Constants.ERROR_ADDED_TO_STATUS in message:
             return
 
-        formatted_message = self.__ensure_error_message_restriction_compliance(message)
         # Compose error detail
-        error_detail = {
-            "code": str(error_code),
-            "message": str(formatted_message)
-        }
+        error_detail = self.__set_error_detail(error_code, message)
 
         # determine if a current operation override has been requested
         current_operation = self.__current_operation if current_operation_override_for_error == Constants.DEFAULT_UNSPECIFIED_VALUE else current_operation_override_for_error
 
         if current_operation == Constants.ASSESSMENT:
+            print('did this get called', current_operation)
```

Comment:
Please self-review code that is pushed before asking reviewers to take a look.

Reply:
done